### ISSUE TITLE
Can it be used in comfyui?

### ISSUE THREAD

**Issue by alex13by** [xinsir6/ControlNetPlus#5]
This model may replace a bunch of controlnet models I have on my hard drive, but can it be used directly in comfyui?

**Comment by xinsir6** [comment]
I am working for it. probably I need some time, there is lot of works to do. I will release the promax version(8 control with tile & inpaiting) within 2 weeks, so be patient. I will try to fix the released model to comfyui in this weekend.

**Comment by Kadah** [comment]
It currently works in ComfyUI using the standard ControlNet nodes. While it might not be optimal, it does appear to be able to pick up on the type from the input image well enough to function without explicitly being able to pass the type.

**Comment by xinsir6** [comment]
Yes, there exists some modification in the pipeline scripts to make it work in comfyui. I will try to update it in this weekend.

**Comment by GitGod76** [comment]
> Yes, there exists some modification in the pipeline scripts to make it work in comfyui. I will try to update it in this weekend.

can it be used in SD A1111?

**Comment by YuanGYao** [comment]
It currently works with ComfyUI and A1111, but it doesn't perform very well. 
When using the example images from this project as control images, it performs well even without a control type id and seems to recognize the control type automatically. 
But when using other images as control images, it starts to perform poorly. The most typical one is OpenPose, and most of the time the generated image will contain the OpenPose skeleton.

**Comment by GitGod76** [comment]
> It currently works with ComfyUI and A1111, but it doesn't perform very well. When using the example images from this project as control images, it performs well even without a control type id and seems to recognize the control type automatically. But when using other images as control images, it starts to perform poorly. The most typical one is OpenPose, and most of the time the generated image will contain the OpenPose skeleton.

Dude how to use it at A1111?

**Comment by 3blackbar** [comment]
https://github.com/Mikubill/sd-webui-controlnet/discussions/2989
